packages/orchestration/src/utils/chainHub.js

```diff
@@ -16,6 +16,20 @@ export const CHAIN_KEY = 'chain';
 /** namehub for connection info */
 export const CONNECTIONS_KEY = 'chainConnection';
 
+/**
+ * Character used in a connection tuple key to separate the two chain ids. Valid
+ * because a chainId can contain only alphanumerics and dash.
+ *
+ * It was chosen to be:
+ *
+ * - easy to read
+ * - common for expressing a tuple
+ * - meaningless on the command line
+ *
+ * @see {@link https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md}
```

Comment:
:1st_place_medal: 
thanks for the research and for citing external constraints in a doc comment